### DISCUSSION TITLE
Extracting the thermal history (Temperature vs. time) for the entire mesh or at least one node

### DISCUSSION THREAD

**Discussion by kakapsu** [ansys/pyadditive#297] [Ideas]
I am trying to extract the temperature for each time step whenever I run thermal analysis simulation. The default option only allows for coaxial sensor which outputs temperatures of an entire layer for one time step. Since the thermal solver solves for temperature over time, I want to record these values to a text/.csv file. I am not capable of implementing this idea because I don't have access to the API and therefore I am not sure which variable names to chose in addition to having no idea how the simulate function works. Do you know if there's a way to either extract the entire  temperature vs time history or at least how to get access to the API(actual API and not the gPCR files)

**Comment by pkrull-ansys** [comment]
@kakapsu There is currently not a way to extract the entire temperature vs time history for the thermal history simulation. To date, it's not been available due to the large amount of data that's generated. However, I've forwarded your request to the solver team.
